packages/realm-web/src/Fetcher.ts

```diff
@@ -194,27 +203,31 @@ export class Fetcher implements LocationUrlContext {
   private readonly transport: NetworkTransport;
   private readonly userContext: UserContext;
   private readonly locationUrlContext: LocationUrlContext;
+  private readonly _extraHeaders?: Headers;
```

Comment:
I would remove the `_` for the `extraHeaders` variable.  One could argue that `_` be used a prefix for private variables, but in that case it should be done everywhere and be a separate PR.

Also, I think the prefix `extra` could also be eliminated.

Reply:
no quarrel with any of that

last push has been rebased onto today's master

Reply:
seems the `_` is still there?

Reply:
For "reasons" switched _my_ default branch to "main", also tripping out on the four-space two-space indent revolution between the branch I'm using here locally (based on realm-web-v1.3.0) and the one I'm pushing to you.  Anyway, I think this problem is addressed correctly now.